# HG changeset patch
# User Torsten <ttl@justmail.de>
# Date 1425643015 -3600
#      Fri Mar 06 12:56:55 2015 +0100
# Node ID 2b5cbea8f442b61aa0cc40fb088980dc109971af
# Parent  bb1629e15afeb12d67966985206c3171da73703b
fix missing close icon of dock widgets in windows (bug #44441)

* octave-dock-widget.cc (set_style): fix bug with setting the close icon name

diff --git a/libgui/src/octave-dock-widget.cc b/libgui/src/octave-dock-widget.cc
--- a/libgui/src/octave-dock-widget.cc
+++ b/libgui/src/octave-dock-widget.cc
@@ -356,18 +356,18 @@ octave_dock_widget::set_style (bool acti
 
 #if defined (Q_OS_WIN32)
   _title_widget->setStyleSheet (css);
   css_button = QString ("background: transparent; border: 0px;");
   _dock_button->setStyleSheet (css_button);
   _close_button->setStyleSheet (css_button);
   _dock_action->setIcon (QIcon (":/actions/icons/" + dock_icon + icon_col +
                                 ".png"));
-  _close_action->setIcon (QIcon (":/actions/icons/widget-close" + dock_icon +
-                                 icon_col + ".png"));
+  _close_action->setIcon (QIcon (":/actions/icons/widget-close" + icon_col +
+                                 ".png"));
 #else
   setStyleSheet (css);
 #endif
 }
 
 void
 octave_dock_widget::handle_settings (const QSettings *settings)
 {
